pkg/resources/spark/statefulset_test.go

```diff
@@ -553,4 +562,112 @@ func testCommonFeatures(t *testing.T, comp Component) {
 		_, err := NewStatefulSet(rc, comp)
 		require.Error(t, err)
 	})
+
+	t.Run("framework config", func(t *testing.T) {
```

Comment:
Go will render subtest names using underscores (i.e. `framework_config`) so it's probably worth just writing the name that way in the first place.